txbits/app/wallet/Wallet.scala

```diff
@@ -304,6 +358,7 @@ class Wallet(rpc: JsonRpcHttpClient, currency: CryptoCurrency, nodeId: Int, para
   }
 
   private def getBalance = {
+    // This method is unused as balance is now tracked in the database
```

Comment:
Adjusting the balance using getbalance would problematic. The balance in the database is updated as we confirm deposits and send withdrawals. The balance reported by getbalance would include deposits that we have not yet confirmed.